features/evaluations.mdx

```diff
@@ -11,11 +11,15 @@ Inspect provides many built-in components, including facilities for prompt engin
 
 In order to run evaluations, you need to have a dataset that has been annotated with Langtrace. To annotate a dataset, you can follow the steps in the [Annotate & Measure](/features/annotations) guide. Once you have an annotated dataset, follow the steps below to run evaluations.
 
+<Note>
+Currently langtrace doesn't support `inspect-ai` versions greater than `0.3.14`
+</Note>
+
 1. Install Langtrace SDK & Inspect AI
 
 ```
 pip install langtrace-python-sdk
-pip install inspect-ai
+pip install inspect-ai==0.3.14
```

Comment:
could you add a note and link to https://github.com/UKGovernmentBEIS/inspect_ai/issues/135

Reply:
so ppl know why this specific version

Reply:
done